src/datetime.js

```diff
@@ -13,7 +13,7 @@ engine.now = function(){
     // would not make a difference for computation, but if the end result of an
     // expression is "now()", then it would look different when output to a user.
     // Construct it ourselves to preserve timezone
-    var now = new Date();
+    var now = constants.nowDate; // a JS Date
     var isoStr = FP_DateTime.isoDateTime(now);
     constants.now = new FP_DateTime(isoStr);
```

Comment:
Down the road, it might make sense to encapsulate the caching within the constants.js, and perhaps, again, giving it a more intuitive name. People have typical expectations on what might be in a file called "constants". It's more like the moment of execution, or something along those lines.